package.json

```diff
@@ -55,7 +56,7 @@
     "npm-run-all": "4.1.5",
     "ts-jest": "^29.1.1",
     "ts-patch": "^3.1.2",
-    "typescript": "^5.3.3",
+    "typescript": "5.3",
```

Comment:
@JetyAdam I think this was unintentional. Can we revert back to ^5.3.3?

Reply:
Reverted